community/README.md

```diff
@@ -0,0 +1,18 @@
+![Logo](https://storage.googleapis.com/model_garden_artifacts/TF_Model_Garden.png)
+
+# TensorFlow Community Models
+
+We provide a curated list of the GitHub repositories with machine learning models and implementations powered by TensorFlow 2.
+
+Unlike our "[official](https://github.com/tensorflow/models/tree/master/official)" and "[research](https://github.com/tensorflow/models/tree/master/research)" repositories, we do not maintain the GitHub repositories listed here. Their respective maintainers support them.
```

Comment:
Could we clarify what are "we"? TensorFlow Model Garden team or TensorFlow or Google teams?

Reply:
Good point. How about changing "We" to "This community repository"?

Reply:
Maybe tensorflow/models owners

Reply:
I replaced "We" with "This repository" since this expression is commonly used.